packages/ses/scripts/hermesc.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+OS="$(uname -s)"
+
+case "$OS" in
+    Linux*)
+        OS_DIR="linux64-bin"
+        ;;
+    Darwin*)
+        OS_DIR="osx-bin"
+        ;;
+    CYGWIN*|MINGW*|MSYS*)
+        OS_DIR="win64-bin"
+        ;;
+    *)
+        echo "Unsupported OS: $OS"
+        exit 1
+        ;;
+esac
+
+HERMESC="../../node_modules/hermes-engine-cli/$OS_DIR/hermesc"
+
+$HERMESC -emit-binary -out test/hermes-smoke.hbc test/hermes-smoke.js
```

Comment:
`concat` SES shim here

Reply:
done in https://github.com/leotm/endo/pull/1/commits/a883b73f0dbd7e54d54ad8abdc1de8c8a4f28335